kuma/settings/common.py

```diff
@@ -959,17 +959,17 @@ def pipeline_one_scss(slug, **kwargs):
     },
     'payments': {
         'source_filenames': (
-            'js/payments-handler.js',
-            'js/payments-faq.js',
+            'js/components/payments/payments-handler.js',
```

Comment:
Don't you need to add the new payments-handler-utils.js file here?

Reply:
Or maybe I just don't understand our js packaging scheme.

Reply:
Nope, nope, you are correct. Doh!